packages/fixtures/boards/make-manifest

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+//utility for creating manifest.json files for the test boards
```

Comment:
Can we leave this out for now? Requires a little extra discussion. The layer type and file type in the manifest is functioning as inputs to tests of whats-that-gerber and gerber-parser, so I'm a little uncomfortable with a manifest generation script that uses whats-that-gerber to generate the expectation

Reply:
Sure, I'll remove that commit for now. The idea is to check over and modify the manifest after it's created and just use the generated one as a base. Maybe I could rename it to `init-manifest`? It's a bit like making a snapshot as well and it should be fairly easy to spot any errors introduced.

Reply:
Wanna put it in its own PR and we can figure it out there?